File: week2/python/task4.py
def get_number(index):
    cnt = index // 3 * 7
    x = index % 3
    if(x==1):
        cnt = cnt + 4
    elif(x==2):
        cnt = cnt + 8
    print (cnt)
    
get_number(1) # print 4
get_number(5) # print 15
get_number(10) # print 25
get_number(30) # print 70

File: week2/javascript/task4.js
function getNumber(index) {
let cnt = Math.floor(index / 3) * 7;
    let x = index % 3;
    if (x === 1) {
        cnt += 4;
    } else if (x === 2) {
        cnt += 8;
    }
    console.log(cnt);
}
getNumber(1);  // print 4
getNumber(5);  // print 15
getNumber(10); // print 25
getNumber(30); // print 70
